compat/suite/green-goo-dao-flow-utils.yaml

```diff
@@ -0,0 +1,8 @@
+description: Green Goo Dao flow-utils
+maintainers:
+  - bastian.mueller@flowfoundation.org
+url: https://github.com/turbolent/flow-utils.git
+branch: improvements
```

Comment:
This is still using a fork, because the repo currently uses `flow-c1`. I upstreamed the changes in https://github.com/green-goo-dao/flow-utils/pull/33